pttrack/static/pttrack/css/bootstrap-theme.min.css

```diff
@@ -2,4 +2,477 @@
\ No newline at end of file
+ */
```

Comment:
I'm pretty surprised you have changes here to the minified bootstrap code. Is this just your editor being a bit enthusiastic about formatting things?